docs/esa/jwst/jwst.rst

```diff
@@ -422,98 +406,71 @@ Query without saving results in a file:
 
   >>> from astroquery.esa.jwst import Jwst
   >>>
-  >>> job=Jwst.launch_job("SELECT TOP 100 \
-  >>> instrument_name, observationuri, planeid, calibrationlevel, \
-  >>> dataproducttype \
-  >>> FROM jwst.main ORDER BY instrument_name, observationuri")
+  >>> job = Jwst.launch_job("SELECT TOP 100 "
+  >>>       "instrument_name, proposal_id, calibrationlevel, "
+  >>>       "dataproducttype "
+  >>>       "FROM jwst.main ORDER BY instrument_name, observationuri")
```

Comment:
This won't work in this format, continued lines need the `...` instead of the `>>>` and I would also consider adding a proper indent to help reading.
```suggestion
  >>> job = Jwst.launch_job("SELECT TOP 100 "
  ...                       "instrument_name, proposal_id, calibrationlevel, "
  ...                       "dataproducttype "
  ...                       "FROM jwst.main ORDER BY instrument_name, observationuri")
```

Reply:
Fixed